data/amharic/bbc_amharic_radio/liveradio.json

```diff
@@ -25,7 +30,7 @@
         "type": "heading"
       },
       {
-        "text": "ዝግጅቶቻችንን",
+        "text": "ዝግጅቶቻችንን’",
```

Comment:
Just wanted to confirm that the " ' " is meant to be there since I saw a removed " . " from somewhere above. Otherwise, LGTM.

Reply:
I noticed this but could not find the translation on the sheet to check. Do we know where the translation for this text is documented?

Reply:
![Image 01-10-2019 at 12 29](https://user-images.githubusercontent.com/54575620/65958447-4e5c6600-e447-11e9-815e-5efeb0b848c7.jpg)
In the translations sheet the apostrophe is present

Reply:
We've automated pulling the translations from the google sheet, so they should match what editorial provide exactly